UPGRADE-3.0.md

```diff
@@ -37,3 +37,59 @@
 - The signature of the `HttpClientFactory::__construct()` method changed to accept instances of the PSR-17 factories in place of Httplug's ones
 - The signature of the `DefaultTransportFactory::__construct()` method changed to accept instances of the PSR-17 factories in place of Httplug's ones
 - The signature of the `GzipEncoderPlugin::__construct()` method changed to accept an instance of the `Psr\Http\Message\StreamFactoryInterface` interface only
+- Removed the `TagsContext`, `ExtraContext` and `Context` classes, data is now stored in a plain array
```

Comment:
`TagContext` no longer does merge, always overrides right?

Reply:
`TagsContext` used to provide a `merge()` function, but now both the `Event` and `Scope` classes provide a setter that imho is quite self-explanatory. Of course when applying a scope on an event the tags still get merged